units/hybrids_of_steel.cfg

```diff
@@ -71,7 +71,7 @@
         number=2
     [/attack]
 [/unit_type]
-[unit_type]
+[unit_type]  # A Spearman_steel advances directly to this (jumping level 4), but it's not much up an upgrade at all.
```

Comment:
I changed this in another branch.  Can I remove this file from the commit?

Reply:
Yes, but it's quite complicated. I think the easiest way to remove this is to erase that comment and include it in a commit in this feature branch.